config.json.example

```diff
@@ -0,0 +1,5 @@
+{
+  "host": "0.0.0.0",
+  "port": 5090,
+  "fingerprint_random_path": "/xilriws-fingerprint-random/"
```

Comment:
is this setting needed? as it's hardcoded in Dockerfile anyway?! @ccev

Reply:
This is required in case people want to run locally